tools/update-github-project-board/index.ts

```diff
@@ -0,0 +1,277 @@
+import { Octokit } from "@octokit/rest";
+import { graphql } from "@octokit/graphql";
+
+const ORG = process.env.ORGANIZATION || "hashicorp";
+const PROJECT_ID = parseInt(process.env.PROJECT_NUMBER || "125", 10);
+type Awaited<T> = T extends PromiseLike<infer U> ? U : T;
+type Issue = Awaited<ReturnType<typeof octokit.issues.list>>["data"][0];
+
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN,
+});
+
+const gql = graphql.defaults({
+  headers: {
+    authorization: `token ${process.env.GITHUB_TOKEN}`,
+  },
+});
+
+async function allRepos() {
+  const repos = ["terraform-cdk"];
+  const allHashicorpRepos = await octokit.paginate(octokit.repos.listForOrg, {
+    org: "hashicorp",
+  });
+
+  allHashicorpRepos.forEach((repo) => {
+    if (repo.name.startsWith("cdktf")) {
```

Comment:
Maybe query by tag instead?
These repos might also be interesting for us but do not share that prefix:
https://github.com/hashicorp/terraform-cdk-go
https://github.com/hashicorp/docker-on-aws-ecs-with-terraform-cdk-using-typescript
https://github.com/hashicorp/kubernetes-on-gcp-with-terraform-cdk